sql/views/pgbouncer_fdw_views.sql

```diff
@@ -1,291 +1,181 @@
+CREATE VIEW @extschema@.pgbouncer_version AS
+    SELECT version_major
```

Comment:
Doesn't this need the name of the pgbouncer in question?

Reply:
Yup! Don't use it internally, but would definitely be helpful for a user calling it.